tardis/montecarlo/packet_source.py

```diff
@@ -164,25 +169,21 @@ def create_packets(
 
         Parameters
         ----------
-        temperature : float64
+        temperature : astropy.Quantity
             Absolute Temperature
         no_of_packets : int
             Number of packets
-        radius : float64
+        radius : astropy.Quantity
             Initial packet radius
-        time_explosion: float64
+        time_explosion: astropy.Quantity
             Time elapsed since explosion
 
         Returns
         -------
-        array
-            Packet radii
-        array
-            Packet frequencies
-        array
-            Packet directions
-        array
-            Packet energies
+        astropy.Quantity : Packet radii
+        astropy.Quantity : Packet frequencies
+        astropy.Quantity : Packet directions
+        astropy.Quantity : Packet energies
         """
```

Comment:
I wonder if these are according to our docstring standards- it's the name of the variable and then the variable's type, right? The next line would have more description of the variable.

Reply:
@atharva-2001 For parameters of the function, the docstring convention is `variable_name (type): description`
However, for return values (like the above lines), the convention is `type: description` (because return values aren't strictly associated with a variable_name)